cmd/reissuer/main.go

```diff
@@ -414,38 +434,60 @@ func reissueCerts(c *cli.Context) (err error) {
-	}
-	if err = sm.With(certreq.Id).AddSecretVersion(ctx, secretType, []byte(pkcs12password)); err != nil {
-		return cli.Exit(fmt.Errorf("could not create password version: %s", err), 1)
-	}
+	if !certreq.NoEmailDelivery {
+		// If email delivery is enabled, create the whisper link to send via email
+		var sm *secrets.SecretManager
+		if sm, err = secrets.New(conf.Secrets); err != nil {
+			return cli.Exit(fmt.Errorf("could not connect to secret manager: %s", err), 1)
+		}
 
-	// Create a Whisper link for the provided PKCS12 password.
-	if whisperLink, err = whisper.CreateSecretLink(fmt.Sprintf(whisperPasswordTemplate, pkcs12password), "", 3, weekFromNow()); err != nil {
-		return cli.Exit(err, 1)
-	}
+		secretType := "password"
 
-	// Create the email manager.
-	if emailer, err = emails.New(conf.Email); err != nil {
-		return cli.Exit(err, 1)
-	}
+		if err = sm.With(certreq.Id).CreateSecret(ctx, secretType); err != nil {
+			return cli.Exit(fmt.Errorf("could not create password secret: %s", err), 1)
+		}
+		if err = sm.With(certreq.Id).AddSecretVersion(ctx, secretType, []byte(pkcs12password)); err != nil {
+			return cli.Exit(fmt.Errorf("could not create password version: %s", err), 1)
+		}
 
-	// Send the notification email that certificate reissuance is forthcoming and provide whisper link to the PKCS12 password.
-	if nsent, err = emailer.SendReissuanceStarted(vasp, whisperLink); err != nil {
-		return cli.Exit(err, 1)
+		// Create a Whisper link for the provided PKCS12 password.
+		if whisperLink, err = whisper.CreateSecretLink(fmt.Sprintf(whisperPasswordTemplate, pkcs12password), "", 3, weekFromNow()); err != nil {
+			return cli.Exit(err, 1)
+		}
+
+		// Create the email manager.
+		if emailer, err = emails.New(conf.Email); err != nil {
+			return cli.Exit(err, 1)
+		}
+
+		// Send the notification email that certificate reissuance is forthcoming and provide whisper link to the PKCS12 password.
+		if nsent, err = emailer.SendReissuanceStarted(vasp, whisperLink); err != nil {
+			return cli.Exit(err, 1)
+		}
+
+		fmt.Printf("successfully sent %d Whisper password notifications for PKCS12 password %q\n", nsent, pkcs12password)
 	}
 
-	fmt.Printf("successfully sent %d Whisper password notifications for PKCS12 password %q\n", nsent, pkcs12password)
+	if certreq.Webhook != "" {
+		// Store the password using the webhook
+		var client api.CourierClient
+		if client, err = api.New(certreq.Webhook); err != nil {
+			return cli.Exit(fmt.Errorf("could not create courier client: %s", err), 1)
+		}
+
+		// Store the password with the webhook
+		req := &api.StorePasswordRequest{
+			ID:       certreq.Id,
+			Password: pkcs12password,
+		}
+		if err = client.StoreCertificatePassword(ctx, req); err != nil {
+			return cli.Exit(fmt.Errorf("could not store password with webhook: %s", err), 1)
+		}
+
+		fmt.Printf("successfully sent PKCS12 password to webhook %q\n", certreq.Webhook)
```

Comment:
Would prefer the verb "send" or "deliver" rather than "store" -- I think this is what might have created the confusion with Google Secret manager above. 

```suggestion
		// Create a courier client to deliver the pkcs12 password to the TRISA member
		var client api.CourierClient
		if client, err = api.New(certreq.Webhook); err != nil {
			return cli.Exit(fmt.Errorf("could not create courier client: %s", err), 1)
		}

		// Deliver the pkcs12 password via the webhook
		req := &api.StorePasswordRequest{
			ID:       certreq.Id,
			Password: pkcs12password,
		}
		if err = client.StoreCertificatePassword(ctx, req); err != nil {
			return cli.Exit(fmt.Errorf("could not deliver pkcs12 password with webhook: %s", err), 1)
		}

		fmt.Printf("successfully sent PKCS12 password to webhook %q\n", certreq.Webhook)
```

Reply:
I think deliver makes more sense to me, thanks for the suggestion!